remote/webby/webby_unix.h

```diff
@@ -19,7 +19,11 @@ static int wb_socket_error(void)
 
 static int wb_valid_socket(webby_socket_t socket)
 {
+#if !defined(__NN_AARCH64__)
   return socket > 0;
+#else
+  return socket >= 0;
+#endif
```

Comment:
I think your original solution was correct because this also works on PS4:
```suggestion
  return socket >= 0;
```